src/static/stilyagi/assets/styles/pages/design.css

```
@@ -0,0 +1,489 @@
+}
+.src-line .gut {
+  color: var(--ink-soft);
+  user-select: none;
+  flex: 0 0 28px;
+  text-align: right;
+  font-size: 0.78rem;
+  padding-top: 1px;
+}
+.src-line .content {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)